user-tools/README-CVText.md

```diff
@@ -0,0 +1,114 @@
+  - `actions` - The list of mouse actions to take at the center point of the returned `rect`.
+
+## Example Segment JSON
+
+### `endCriteria`
+
+```json
+{
+    "name":"CV Text Criteria: Create New Profile Button Text",
+    "sessionId":"12345",
+    "endCriteria":[
+        {
+            "type":"CVText",
+            "transient":true,
+            "data":{
+                "text":"CREATE NEW PROFILE",
+                "textMatchingRule":"Contains",
+                "textCaseRule":"Ignore"
+                "withinRect": {
+                    "screenSize":{"x": 1656, "y": 724},
+                    "rect":{"x": 900, "y": 110, "width": 350, "height": 50}
+                } 
+            }
+        }
+    ],
+    "botAction":{}
+}
+```
+
+### `botAction`
+
+```json
+{
+    "name": "CV Text Action: Click Create New Profile Button",
+    "description":"Moves the mouse over the Create New Profile button text, then clicks and releases on the button. Criteria waits for the action to complete.",
+    "sessionId":"67890",
+    "endCriteria": [
+        {"type":"ActionComplete" }
+    ],
+    "botAction":{    
+        "type":"Mouse_CVText",
+        "data": {
+            "type":"CVText",
+            "transient":true,
+            "data":{
+                "text":"CREATE NEW PROFILE",
+                "textMatchingRule":"Contains",
+                "textCaseRule":"Ignore"
+                "withinRect": {
+                    "screenSize":{"x": 1656, "y": 724},
+                    "rect":{"x": 900, "y": 110, "width": 350, "height": 50}
+                } 
+            }
+            "actions": [
+                {"leftButton":false,"middleButton":false,"rightButton":false,"forwardButton":false,"backButton":false,"scroll":{"x":0.0,"y":0.0},"duration":2.0 },
+                {"leftButton":true,"middleButton":false,"rightButton":false,"forwardButton":false,"backButton":false,"scroll":{"x":0.0,"y":0.0} },
+                {"leftButton":false,"middleButton":false,"rightButton":false,"forwardButton":false,"backButton":false,"scroll":{"x":0.0,"y":0.0} }
+            ]
+        }
+    }
+}
+```
+
+## Demos
+**TODO:** Link to a public git repo files that are using this feature. **Feel free to skip this as I might be making a lot of these myself.**
+- `endCriteria` - (Non-Public) https://github.com/Regression-Games/RGBossRoom/tree/main/Assets/RegressionGames/Resources/BotSegments/computer_vision/text_discover_criteria_menu_changeprofilebutton
+- `botAction` - (Non-Public) https://github.com/Regression-Games/RGBossRoom/tree/main/Assets/RegressionGames/Resources/BotSegments/computer_vision/text_discover_action_menu_changeprofilebutton
+
+## Limitations and Notes
+The `CVText` type is still in an experimental phase and may provide inconsistent or unexpected results in some situations.
+- `textCaseRule` must always be set to `Ignore`.  The AIService does not currently consider capitalization in its results.
```

Comment:
The user doesn't know about AIService so I'd say that `textCaseRule` is just currently unsupported

Reply:
i'll update this line accordingly

Reply:
Actually, I think I'll leave it, AIService as a concept is discussed throughout this doc as an external service.